index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <header>
+     <div class="welcome">
+        <h1>Hi there</h1>
+     </div>
+     <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+     </div>
+   </header> 
+   <section class="dashboard">
```

Comment:
- [ ] This section has a background image. Kindly include it. 

![image](https://user-images.githubusercontent.com/84156714/212361605-4f8412a5-391d-4463-b4a0-541f8a4f60d9.png)